easybuild/easyconfigs/c/cget/cget-0.1.9-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,47 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+# 
+# Author: Jonas Demeulemeester
+# The Francis Crick Insitute, London, UK
+# Modified for 0.1.9 by Adam Huffman, Big Data Institute, University of Oxford
+##
+easyblock = 'PythonBundle'
+
+name = 'cget'
+version = '0.1.9'
+
+homepage = 'https://cget.readthedocs.io/en/latest/index.html'
+description = """Cmake package retrieval. This can be used to download and install cmake packages"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+builddependencies = [('binutils', '2.35')]
+
+# Since cget uses CMake to install packages, CMake is a runtime dependency
+dependencies = [
+    ('Python', '3.8.6'),
+    ('CMake', '3.18.4'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('click', '8.0.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/c/click/'],
+        'checksums': ['8c04c11192119b1ef78ea049e0a6f0463e4c48ef00a30160c704337586f3ad7a'],
+    }),
+    ('six', '1.16.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/six/'],
+        'checksums': ['1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926'],
+    }),
```

Comment:
Actually, i just noticed these dependencies. It's fine if we want to keep these newer versions, but I suspect maybe the exact version of these isn't very important, if so, they are nowdays included in the Python bundle.

Reply:
@verdurin ping?

Reply:
Yes, according to https://github.com/pfultz2/cget/blob/master/requirements.txt the Python bundle versions of those extensions are new enough.

Would you prefer that I remove them from this bundle, then?

Reply:
Absolutely, yes, if the `click` and `six` that come with the Python dependency suffice, there's no reason to include specific version here.